junit-vintage-engine/src/test/java/org/junit/vintage/engine/samples/junit4/ChildlessTestClass.java

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright 2015-2017 the original author or authors.
+ *
+ * All rights reserved. This program and the accompanying materials are
+ * made available under the terms of the Eclipse Public License v1.0 which
+ * accompanies this distribution and is available at
+ *
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+
+package org.junit.vintage.engine.samples.junit4;
+
+import org.junit.runner.RunWith;
+import org.junit.vintage.engine.VintageTestEngineExecutionTests;
+
+@RunWith(VintageTestEngineExecutionTests.MisbehavingChildlessRunner.class)
+public class ChildlessTestClass {
```

Comment:
Why did you make this a top-level class instead of simply a nested static class in the test class that actually uses it?